templates/staffing/prod_report.html

```diff
@@ -42,38 +42,53 @@
 </div>
 
 <div class="row"><br/></div>
-
 <div class="row">
     <div class="col-md-12">
     <div class="table-responsive">
-
     <table class="table table-striped table-hover table-pydici-extra-condensed" id="prod_table">
 
     <tr>
         <th></th>
     	{% for month in months %}
-    	    <th colspan="2" style="border-left:1px solid #C3C3C3;"><center>{{ month|date:"F y" }}</center></th>
+        {% if month.month == today.month and month.year == today.year %}
```

Comment:
Current month is always last month. So you could use forloop.last to check that in a cleaner way.
See : https://docs.djangoproject.com/fr/1.8/ref/templates/builtins/#for